operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Container.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+import java.util.Optional;
+
+import oracle.kubernetes.json.Description;
+import oracle.kubernetes.json.EnumClass;
+import oracle.kubernetes.operator.ImagePullPolicy;
+import oracle.kubernetes.operator.helpers.KubernetesUtils;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class Container {
+
+  public static final String INIT_CONTAINER_NAME_PREFIX = "operator-common-container";
+  public static final String DEFAULT_INIT_CONTAINER_COMMAND = "cp -R $COMMON_MOUNT_PATH/* $COMMON_TARGET_PATH";
+  public static final String INIT_CONTAINER_WRAPPER_SCRIPT = "/weblogic-operator/scripts/commonMount.sh";
```

Comment:
It would help IMO to have this script avoid doing an 'exit 1' (as exit 1 handling is known to be problematic at the Operator level), and to have this script record its output so the main introspector pod log, plus the pod logs for every weblogic pod, can include its tracing in its stdout


```
/weblogic-operator/scripts/commonMount.sh > /tmp/cm.out 2>&1 ; cat /tmp/cm.out ; cp /tmp/cm.out  $COMMON_MOUNT_DIR/commonMountLogs/commonMount-1.out ; exit 0
```
in addition the wl pods (and probably also the introspector pod script), should sanity check the script output to make sure nothing went wrong. Particularly as the operator is not involved with checking the results for each wl pod.

Reply:
If the ^^^ command is too complex, the script itself could be adjusted to do the same steps - which might be more elegant. (You'll need to pass in teh container# to use for decorating the output.)

Reply:
If the init container fails for WebLogic pods, pods will be in `Init:Error` status, and users can always get the logs of the init container using `kubectl logs -c` command. So it's not an issue for WebLogic pods.For introspector pod, I'm are copying the output of script in operator logs and if the introspector pod is still around, users can get the logs using `kubectl logs`. I think it would be good to discuss this in a meeting since this was missed during the original design/implementation meeting.

Reply:
I have made the changes to capture the output of wrapper script in the main introspector pod log and the pod logs for every weblogic pod. Please let me know if missed something.